enterprise/server/remote_execution/platform/platform.go

```diff
@@ -445,6 +453,49 @@ func int64Prop(props map[string]string, name string, defaultValue int64) int64 {
 	return i
 }
 
+func iecBytesProp(props map[string]string, name string, defaultValue int64) int64 {
+	val := props[strings.ToLower(name)]
+	if val == "" {
+		return defaultValue
+	}
+	// If it looks like a float (e.g. 1e9), convert to an integer number of
+	// bytes.
+	f, err := strconv.ParseFloat(val, 64)
+	if err == nil {
+		return int64(f)
+	}
+	// Otherwise attempt to parse as an IEC number of bytes.
+	// Example: 4GB = 4*(1024^3) bytes
+	n, err := units.RAMInBytes(val)
+	if err != nil {
+		return defaultValue
+	}
```

Comment:
I'm cool with this as-is, but we should have a way to indicate to users that their buildbuddy.yaml file is misconfigured, rather than always failing silently and using a default value (which could be really frustrating for a user trying to figure out why something is not having an effect)

Reply:
I'll add a TODO here for that -- shouldn't be too hard but will require a bit of refactoring.